src/config.cpp

```diff
@@ -185,6 +185,14 @@ void parse_program_options(int argc, const char** argv)
         .metavar("PID")
         .optional();
 
+    general_options.toggle("as-pre-sudo-user", "Drop root privileges before launching COMMAND.")
+        .short_name("u");
+
+    general_options.option("as-user", "User to drop privileges to before launching COMMAND.")
```

Comment:
I think this is easier to understand:
"Launch the COMMAND as the user USERNAME"